spec/unit/coinbase/wallet_spec.rb

```diff
@@ -98,7 +118,7 @@
   describe '.create' do
     let(:wallet_id) { SecureRandom.uuid }
     let(:create_wallet_request) do
-      { wallet: { network_id: network_id } }
+      { wallet: { network_id: network_id, use_server_signer: false } }
```

Comment:
We should test wallet creation when using a server signer and ensure this gets passed with a true value

Reply:
this was missed in the rebase. added it back